ci-operator/step-registry/ipi/conf/vsphere/zones/multisubnets/ipi-conf-vsphere-zones-multisubnets-ref.yaml

```diff
@@ -2,6 +2,11 @@ ref:
   as: ipi-conf-vsphere-zones-multisubnets
   from: upi-installer
   commands: ipi-conf-vsphere-zones-multisubnets-commands.sh
+  env:
+  - name: LB_TYPE
+    default: OpenShiftManagedDefault
```

Comment:
If I remember right, the `ipi-conf-vsphere-zones-multisubnets` is used for `cucushift-installer-rehearse-vsphere-ipi-zones-multisubnets-external-lb`, which means it using external lb, and leave `apiVIPs` and `ingressVIPs` to be empty on purpose, I feel like your case deserve a separate install workflow, but not using the existing one.

Reply:
Yes, there should be 2 ways for now to setup cluster with ELB for now
1. set apiVIPS and ingressVIPs empty and do not set lb_type
2. Set apiVIPS and ingressVIPs with lb_type is usermanaged. for feature https://issues.redhat.com/browse/OPNET-305

using the existing one is easy to implement 2.